site/layouts/shortcodes/orange-footer.html

```diff
@@ -0,0 +1,172 @@
+{{- /*
+  Usage: `orange-footer args`
+
+  `args` are all optional and can be one of the following:
+    * mailing: Display mailing content - default = false
+    * follow: Display follow content - default = false
+    * nav: Display nav content - default = false
+    * service: Display service content - default = false
+    * mandatory: Display mandatory content - default = false
+*/ -}}
+
+{{- $mailing := .Get "mailing" | default false -}}
+{{- $follow := .Get "follow" | default false -}}
+{{- $nav := .Get "nav" | default false -}}
+{{- $service := .Get "service" | default false -}}
+{{- $mandatory := .Get "mandatory" | default false -}}
+{{- $id := .Get "id" -}}
+{{- $input := .Inner -}}
+
+<footer class="footer bg-dark navbar-dark">
+  <h2 class="visually-hidden">Site map & information</h2>
+  {{ if $mailing }}
+  <!-- Mailing list -->
+  <div class="container footer-title-content">
+    <h3 class="footer-heading">Sign up to our mailing list</h3>
+    <div class="row">
+      <form class="d-flex col-12 col-md-9 col-lg-7 col-xl-6 col-xxl-5 gap-2 gap-md-3">
+        <label for="inputEmail{{ $id }}" class="visually-hidden">Email</label>
+        <input type="email" class="form-control bg-dark border-dark text-white" id="inputEmail{{ $id }}" placeholder="Enter your email">
+        <button type="submit" class="btn btn-secondary btn-inverse text-nowrap">Sign up</button>
+      </form>
+    </div>
+  </div>
+  {{ if or $follow $nav $service $mandatory }}
+  <div class="border-bottom border-1 border-dark"></div>
+  {{ end }}{{ end }}
+  {{ if $follow }}
+  <!-- Follow us bar -->
+  <div class="container footer-social">
+    <h3 class="footer-heading">Follow us</h3>
+    <ul class="navbar-nav gap-2 flex-row align-self-start">
```

Comment:
Only 6 social buttons in DSM, normal?
Should they stay on one line only?

Reply:
At first, I wanted to give every button to the developer. As-is, he just needs to remove what's not needed without searching into docs.